src/renderer/assets/styles/reader-app.css

```diff
@@ -2,6 +2,34 @@
 @import "./focus.css";
 @import "./variable.css";
 
+/* #annotation-layer {
+  position: absolute;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  overflow: hidden;
+  opacity: 0.2;
+  line-height: 1.0;
+}
+
+#annotation-layer > section {
+  color: transparent;
+  position: absolute;
+  white-space: pre;
+  cursor: text;
+  transform-origin: 0% 0%;
+}
+
+#annotation-layer > .linkAnnotation > a {
+  position: absolute;
+  font-size: 1em;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+} */
```

Comment:
pdfReader.ts is code executed in sandboxed webview, so this “reader” CSS does not reach inside the DOM created by PDF.js (canvas, etc.)

Reply:
alternative solution: inject styles into webview DOM via preload script, or even more easily, via the EJS HTML template

Reply:
but of course !